### PR TITLE
US906142: Merged opensuse base and direct dependents into one repository

### PR DESCRIPTION
**Pull request by michael-bryson**
Ticket: https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=906142

Set the versions to 4.0.0-SNAPSHOT as that was the first major version not used before by these images.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/target/
+**/target/
```
Comment (dermot-hardy) [inline .gitignore:2]:
```suggestion
target/
```

File: official-build.props [hunk 2]
```diff
@@ -0,0 +1,3 @@
+DISABLE_AUTOTAGGING=true
```
Comment (dermot-hardy) [inline official-build.props:1]: Check with Tony but I don't think the `DISABLE_AUTOTAGGING=true` line is required.

```suggestion
```

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        <module>opensuse-java11-images</module>
+        <module>opensuse-java17-images</module>
+        <module>opensuse-nodejs18-image</module>
+        <module>opensuse-dotnet6-images</module>
+        <module>opensuse-python3-image</module>
+    </modules>
+
+    <properties>
+        <copyrightYear>2024</copyrightYear>
+        <copyrightNotice>Copyright ${project.inceptionYear}-${copyrightYear} Open Text.</copyrightNotice>
```
Comment (dermot-hardy) [inline pom.xml:73]:
```suggestion
```

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+                <plugin>
+                    <groupId>io.fabric8</groupId>
+                    <artifactId>docker-maven-plugin</artifactId>
+                    <version>0.43.4</version>
+                </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-shade-plugin</artifactId>
+                    <version>3.5.1</version>
+                </plugin>
```
Comment (dermot-hardy) [inline pom.xml:93]:
```suggestion
```

**Review by dermot-hardy — APPROVED** [review]

File: official-build.props (hunk 2 above)
Comment (michael-bryson) [reply official-build.props:1]: @tonymcveigh Is this setting required anymore as per Dermots comment?

Comment (tonymcveigh) [reply official-build.props:1]: No, that's an old setting. No longer required.

**Pull request merged**